### PR TITLE
4.13 -long duration cloud cases

### PR DESCRIPTION
**Pull request by miyadav**
@sunzhaohua2 @huali9 @jhou1 PTAL , we missed to add on 4.13 , hence making this PR.
Once this is merged will add slack notification as well.

### CODE REVIEW

**Comment by sunzhaohua2** [comment]
@miyadav Thanks, I think we can use different workflows from 4.12, so it seems that more configurations can be covered.
And some platforms can choose arm cluster, wdyt?

**Comment by miyadav** [comment]
@sunzhaohua2 , yes , just that , if we have similar profiles easy to check back and compare results , will think about it , will definitely add for arm. 
Thanks

**Comment by miyadav** [comment]
@sunzhaohua2  as suggested added for arm64 and replaced two workflows from 4.12 -> 4.13 , PTAL and suggest , thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -962,6 +976,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private
+- as: aws-usgov-ipi-private-sts-p3-f28-long-duration-cloud
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:979]: Job name should be consistent with workflow and end with`p3-f28-long-duration-cloud`, this one should be aws-ipi-disconnected-private-p3-f28-long-duration-cloud

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -375,6 +375,20 @@ tests:
...
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;NonPreRelease&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-aws-ipi-private
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:391]: We already covered workflow `cucushift-installer-rehearse-aws-ipi-disconnected-private`, so I think we can remove this one or add a new one in arm

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -1432,6 +1459,19 @@ tests:
...
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@azure-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-azure-ipi-workers-marketimage
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1474]: This one is same with 4.12, can we change it to `cucushift-installer-rehearse-azure-mag-ipi-fullyprivate` or remove it only keep one in arm.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 4]
```diff
@@ -1803,6 +1843,18 @@ tests:
...
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@gcp-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-techpreview
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1857]: This one is same with 4.12, maybe we can change it to `cucushift-installer-rehearse-gcp-ipi-private`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 5]
```diff
@@ -891,6 +891,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-disconnected
+- as: azure-ipi-disconnected-p3-f28-long-duration-destructive
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:894]: Should be `azure-ipi-disconnected-p3-f28-long-duration-cloud`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 1 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:979]: done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 3 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1474]: done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 4 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1857]: done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 5 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:894]: done

**Comment by miyadav** [comment]
Thanks @sunzhaohua2 for detailed review , have made modifications , PTAL when time permits.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 6]
```diff
@@ -223,6 +223,20 @@ tests:
...
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;NonPreRelease&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-aws-ipi-private
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:239]: This part should be removed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 7]
```diff
@@ -891,6 +919,23 @@ tests:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      E2E_RUN_TAGS: '@arm64 and @azure-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips and @destructive and @long-duration'
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.13'
+      TEST_SCENARIOS: MCO|Cluster_Infrastructure
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:935]: Please add` TEST_TIMEOUT: "90"`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 8]
```diff
@@ -241,6 +255,20 @@ tests:
...
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: aws-ipi-private-p3-f28-long-duration-cloud
+  cron: 28 0 27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;NonPreRelease&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
Comment (sunzhaohua2) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:266]: Don't need `TEST_FILTERS`, it's covered in `chain: openshift-e2e-test-qe-long-duration`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 6 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:239]: sorry just realised it is repeated. Will remove

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 8 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:266]: done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 7 above)
Comment (miyadav) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:935]: done

**Comment by miyadav** [comment]
cc @liangxia , rebased and made changes as suggested , PTAL when possible.

**Pull request merged**